tests/test_file_processing.py

```diff
@@ -16,7 +17,11 @@ def sample_pdf(tmp_path):
 
 def test_pdf_converter(caplog, sample_pdf):
     caplog.set_level(logging.INFO)
+    file_id = "test_file"
+    output_file = f'docs/examples/sciencebeam_xml_outputs/{file_id}.xml'
```

Comment:
see other note on output file. Tests/running the code should not write to the source tree, except perhaps the current directory.